tests/CommonTests.cs

```diff
@@ -173,7 +169,7 @@ public async Task CanCreateInvoiceUsingConnectionString()
 		public async Task CanGetInfo()
 		{
 			await WaitServersAreUp();
-			var blockHeight = Tester.CreateRPC().GetBlockCount();
+			await Tester.CreateRPC().GetBlockCountAsync();
```

Comment:
`we don't use the blockHeight`?? in this case this line is useless.

Reply:
Yeah, I now tried to assert the blockheight in the check below, but it seems to be unreliable (or too much latency in the tests), so I'll remove it.